mariadb/content.md

```diff
@@ -117,7 +117,13 @@ Set to a non-empty value, like `yes`, to generate a random initial password for
 
 ### `MARIADB_ROOT_HOST` / `MYSQL_ROOT_HOST`
 
-This is the hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component).
+This is the hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component). Setting this to `localhost` will prevent any root user being accessible except via the unix socket.
```

Comment:
```suggestion
This is the hostname part of the root user created. By default this is `%`, however it can be set to any default [MariaDB allowed hostname component](https://mariadb.com/kb/en/create-user/#host-name-component). Setting this to `localhost` will prevent root user access from other containers unless a shared unix socket is used.
```

Reply:
I started writing this way first, but then got to the point when I asked "do I need to explain the shared unix socket now?".

Reply:
That's the question :smile: My change mostly targets "any root user", which isn't quite right, thus I changed it to "from other containers". Since nothing special (in terms of "different to a not containerized server") is happening here we might also simply skip the second sentence.